lib/hackney/income/tenancy_prioritiser/tenancy_classification.rb

```diff
@@ -235,7 +235,8 @@ def after_letter_one_actions
             Hackney::Tenancy::ActionCodes::INCOME_COLLECTION_LETTER_2_UH_ALT_2,
             Hackney::Tenancy::ActionCodes::INCOME_COLLECTION_LETTER_2_UH_ALT_3,
             Hackney::Tenancy::ActionCodes::S0A_ALTERNATIVE_LETTER,
-            Hackney::Tenancy::ActionCodes::COURT_WARNING_LETTER_SENT
+            Hackney::Tenancy::ActionCodes::COURT_WARNING_LETTER_SENT,
+            Hackney::Tenancy::ActionCodes::STAGE_THREE_COMPLETE
```

Comment:
I'd like to prepend any _magic UH stages_ with `UH` just so it doesn't seem like our system supports them 
```suggestion
            Hackney::Tenancy::ActionCodes::UH_STAGE_THREE_COMPLETE
```